did.spec

```diff
@@ -18,6 +18,15 @@ BuildRequires: python3-httplib2
 BuildRequires: python3-pytest
 BuildRequires: python3-requests-gssapi
 BuildRequires: python3-setuptools
+# The copr epel 8 builder is complaining about missing modules if this is
+# missing.
```

Comment:
Hm, interesting that this is relevant for `el8` only...

Reply:
It was failing on el8 at least and the rest of builders didn't complain, hence the fenced block.

Reply:
Here's a recent `el8` copr build from #286:

* https://copr.fedorainfracloud.org/coprs/packit/psss-did-286/build/5265676/

I don't see any problems with building there. Is this really needed?

Reply:
I can try to remove them and we'll see.